docs/source/tutorial.ipynb

```diff
@@ -127,15 +139,142 @@
+      "text/plain": [
+       "<Figure size 720x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "from sphviewer.tools import QuickView\n",
     "\n",
+    "fig, ax = plt.subplots(nrows=1, ncols=2, figsize=(10, 5))\n",
+    "\n",
+    "qv = QuickView(pos, mass, r=\"infinity\", plot=False)\n",
+    "img = qv.get_image()\n",
+    "extent = qv.get_extent()\n",
+    "ax[0].imshow(img, extent=extent, vmin=5.5, vmax=8.8)\n",
+    "ax[0].set_xlabel(\"x [kpc]\", fontsize=12)\n",
+    "ax[0].set_ylabel(\"y [kpc]\", fontsize=12)\n",
+    "\n",
+    "qv = QuickView(pos, mass, r=\"infinity\", plot=False, t=90)\n",
+    "img = qv.get_image()\n",
+    "extent = qv.get_extent()\n",
+    "ax[1].imshow(img, extent=extent, vmin=5.5, vmax=8.8)\n",
+    "ax[1].set_xlabel(\"x [kpc]\", fontsize=12)\n",
+    "ax[1].set_ylabel(\"z [kpc]\", fontsize=12)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Properties of `galaxy` object"
+   ]
+  },
```

Comment:
I would change the data visualization part, at the end of everything as a separate section. Because it's like a help how to visualize the data, but it's not a functionality of our API.